www-servers/nginx-unit/nginx-unit-1.3-r1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{3,4,5,6})
+
+inherit python-single-r1
+
+MY_P="unit-${PV}"
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+MY_IUSE="perl python ruby"
+IUSE="${MY_IUSE}"
+REQUIRED_USE="|| ( perl python ruby ) python? ( ${PYTHON_REQUIRED_USE} )"
```

Comment:
Don't you want to reuse MY_IUSE here?

Reply:
I thought about it myself, then decided to use what you wrote verbatim. I can of course change the REQUIRED_USE definition if you like.